omod-common/src/main/java/org/openmrs/module/webservices/rest/web/api/impl/RestHelperServiceImpl.java

```diff
@@ -125,6 +126,16 @@ private Session getSession() {
 		return ret;
 	}
 	
+	@Override
+	public List<Patient> findPatientsByIdentifierStartingWith(String identifier, boolean includeVoided) {
+		Criteria criteria = getSession().createCriteria(Patient.class);
+		criteria.createAlias("identifiers", "identifiers");
+		criteria.add(Restrictions.like("identifiers.identifier", identifier, MatchMode.START));
+		criteria.setResultTransformer(Criteria.DISTINCT_ROOT_ENTITY);
+		criteria.add(Restrictions.eq("voided", includeVoided));
```

Comment:
As it is now if you call it with includeVoided=true only voided patients will be returned, whereas both voided and non voided patients should be returned.